packages/xrpl/src/models/transactions/transaction.ts

```diff
@@ -91,18 +93,20 @@ import {
  * @category Transaction Models
  */
 export type Transaction =
-  | AccountDelete
-  | AccountSet
   | AMMBid
+  | AMMCreate
   | AMMDelete
   | AMMDeposit
-  | AMMCreate
   | AMMVote
   | AMMWithdraw
+  | AccountDelete
+  | AccountSet
```

Comment:
nit: Alphabetically `AccountDelete` and `AccountSet` should be before `AMM...`

Reply:
(Same nit applies below with the case statement if we want to maintain alphabetical order)

Reply:
I was sorting with capitals before lowercase, which makes more sense to me, but if you disagree I'm happy to change that.

Reply:
I personally don't consider capitalization when scanning lists, so I'd prefer to ignore it when sorting alphabetically